cmd/backup-manager/app/util/util.go

```diff
@@ -389,12 +389,17 @@ func GetCommitTsFromBRMetaData(provider v1alpha1.StorageProvider) (uint64, error
 }
 
 // ConstructArgs constructs the rclone args
-func ConstructArgs(conf string, opts []string, command, source, dest string) []string {
+func ConstructArgs(conf string, opts []string, command, source, dest string, rcloneLog bool) []string {
 	var args []string
 	if conf != "" {
 		args = append(args, conf)
 	}
 	if len(opts) > 0 {
+		for _, opt := range opts {
+			if opt == "-q" || strings.HasPrefix(opt, "-v") {
```

Comment:
Also need to consider `--quiet`.

Reply:
No need to check `-v`, just to append them is OK, right?

Reply:
It's ok!

Reply:
Not ok. If user define `-v`, we should filter `-v` for `rclone ls` cmd in fun `getBackupSize` in `cmd/backup-manager/app/export/export.go`, or cmd would like `rclone ls -v` and fun `getBackupSize` will fail. I wil fix this logic in next commit.